### DISCUSSION TITLE
How to stop PDF tearing / glitching on pinch zoom iOS Safari

### DISCUSSION THREAD

**Discussion by box10** [mozilla/pdf.js#17459] [Q&A]
Hi all
I am using the viewer.html to simply display a pdf for safari iOS
It display fine. But when pinch zooming the pdf tears and glitches (see video below)
https://vimeo.com/897571746?share=copy

Is there a fix, or workaround for this?
Manually zooming and panning works fine on iOS safari, just pinch zoom seems a problem.
Any help would be most gratefully appreciated 
Thanks

**Comment by nicolo-ribaudo** [comment]
I opened https://github.com/mozilla/pdf.js/pull/17571 to fix this, with a workaround that you can copy-paste in your website without waiting for it to be fixed here.

Do you have an iPhone and could verify that the proposed fix works there too? :)
